File: dataset/output/GEMINI_gemini-1.5-pro-001/Task125_GEMINI_gemini-1.5-pro-001.c
#include <stdio.h>
#include <string.h>

int main() {
    char userInput[100];

    printf("Enter your choice: ");
    scanf("%s", userInput);

    if (strcmp(userInput, "option1") == 0) {
        printf("Accessing resource 1...
");
    } else if (strcmp(userInput, "option2") == 0) {
        printf("Accessing resource 2...
");
    } else {
        printf("Invalid choice.
");
    }

    return 0;
}

File: dataset/output/GEMINI_gemini-1.5-pro-001/Task125_GEMINI_gemini-1.5-pro-001.java
import java.util.Scanner;

public class ResourceAccess {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your choice: ");
        String userInput = scanner.nextLine();

        if (userInput.equals("option1")) {
            System.out.println("Accessing resource 1...");
        } else if (userInput.equals("option2")) {
            System.out.println("Accessing resource 2...");
        } else {
            System.out.println("Invalid choice.");
        }
    }
}